apps/aragon-fundraising/app/src/screens/Overview.js

```diff
@@ -29,22 +31,74 @@ export default () => {
   // *****************************
   const { reserveBalance, price } = useContext(MainViewContext)
 
+  // *****************************
+  // internal state
+  // *****************************
+  // the batch we use to compute trends
+  const [trendBatch, setTrendBatch] = useState(null)
+
+  // *****************************
+  // effects
+  // *****************************
+  // update the base batch to compute trends
+  useEffect(() => {
+    // search the closest batch from (now - 1 month) to create some trends
+    const oneMonthAgo = subMonths(new Date(), 1).getTime()
+    const trendBatch = batches.reduce(
+      (closest, b) => {
+        const currentClosest = Math.abs(closest.timestamp - oneMonthAgo)
+        const current = Math.abs(b.timestamp - oneMonthAgo)
+        return currentClosest < current ? closest : b
+      },
+      { timestamp: new Date() }
+    )
+    setTrendBatch(trendBatch)
+  }, [batches])
+
   // *****************************
   // human readable values
   // *****************************
-  const adjustedTokenSupply = formatBigNumber(realSupply, tokenDecimals)
-  const adjustedReserves = reserveBalance ? formatBigNumber(reserveBalance.minus(toBeClaimed), daiDecimals) : '...'
-  const adjustedMonthlyAllowance = formatBigNumber(toMonthlyAllocation(rate, daiDecimals), daiDecimals)
-  const adjustedMarketCap = price ? formatBigNumber(price.times(realSupply), daiDecimals) : '...'
-  const adjustedPrice = price ? formatBigNumber(price, 0) : '...'
+  //
+  // numbers
+  const adjustedPrice = price ? formatBigNumber(price, 0, { numberPrefix: '$' }) : '...'
+  const marketCap = price ? price.times(realSupply) : null
+  const adjustedMarketCap = price && marketCap ? formatBigNumber(marketCap, daiDecimals, { numberPrefix: '$' }) : '...'
   const tradingVolume = orders
     // only keep DAI orders
     .filter(o => o.collateral === daiAddress)
     // keep values
     .map(o => o.value)
     // sum them and tada, you got the trading volume
     .reduce((acc, current) => acc.plus(current), new BigNumber(0))
-  const adjsutedTradingVolume = formatBigNumber(tradingVolume, daiDecimals)
+  const adjsutedTradingVolume = formatBigNumber(tradingVolume, daiDecimals, { numberPrefix: '$' })
```

Comment:
Minor typo on all occurrence of `adjsutedTradingVolume` which should be `adjustedTradingVolume`

Reply:
Good catch, changed it